src/domain/signup/utils.ts

```diff
@@ -88,3 +91,15 @@ export const getSignupGroupInitialValuesFromSignup = (
     signups: [getSignupInitialValues(signup)],
   };
 };
+
+export const omitSensitiveDataFromSignupPayload = (
+  payload: SignupInput | UpdateSignupMutationInput
+): Partial<SignupInput | UpdateSignupMutationInput> =>
+  omit(payload, [
```

Comment:
For the back-end exceptions, also these fields were included in the data scrubbing rules:
```
city
street_address
zipcode
membership_number
native_language
service_language
```
These should probably unified.

Reply:
Good point. I unified the data. Also created a PR to Linked Registration UI to make it happen there as well. https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/92